### PR TITLE
Add bleeding column backgrounds to ffe-grid(-react)

### LINKED ISSUE #197: Add support for bleeding grid column backgrounds
#### Which package is this issue related to?
ffe-grid / ffe-grid-react

#### Describe your issue (screenshots welcome!)
Our designs specify a column with a background color - where the background color "bleeds" out into both gutters. This is currently not possible from userland without adding some pretty hefty hacks.

#### What is the requested behavior? 
To be able to specify background colors for columns, and make them fill the surrounding gutters as well. Not sure if we should implement this for small devices too or not.

#### What is the actual behavior?
Currently, there is no built in way to create this behavior from userland without overwriting a ton of CSS rules.

### PR DESCRIPTION
**Pull request by selbekk**
This commit adds support for adding background colors to columns. These background colors will bleed out into their surrounding gutters, to make the content stand out more.

Notice that this is implemented in LESS by simply copying a lot of the `ffe-grid__row--bg-` classes. I considered pulling these out into their own helper classes, but that would've been quite a breaking change for many consumers - without any real gain (well, a few bytes less CSS I guess).

Fixes #197

### CODE REVIEW

File: packages/ffe-grid-react/src/background-colors.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default [
+    'blue-cobalt',
+    'blue-ice',
+    'blue-pale',
+    'blue-royal',
+    'green-mint',
+    'grey-cloud',
+    'purple-magenta',
+    'sand',
+];
```
Comment (wkillerud) [inline packages/ffe-grid-react/src/background-colors.js]: Prettier doesn't add an empty newline at EOF? :police_car:

**Review by wkillerud — DISMISSED** [review]
I'd love some screenshots of the result

File: packages/ffe-grid-react/src/background-colors.js (hunk 1 above)
Comment (selbekk) [reply packages/ffe-grid-react/src/background-colors.js]: Doesn't look like it - but the linter should at least warn. It doesn't though. Anyhow, fixed!

**Review by wkillerud — CHANGES_REQUESTED** [review]
Looks like some old temporary files from ` packages/ffe-icons-react/tmp/*` were commited.

**Comment by selbekk** [comment]
Ah god damn it. I'll make sure to add them to the gitignore file as well

File: packages/ffe-grid-react/src/Grid.md [hunk 2]
```diff
@@ -58,5 +58,37 @@ const backgroundColors = [
...
+        <GridCol sm={12} md={3}>Litt innhold</GridCol>
+        <GridCol sm={12} md={3}>Litt innhold</GridCol>
+        <GridCol sm={12} md={3}>Litt innhold</GridCol>
+        <GridCol sm={12} md={3}>Litt innhold</GridCol>
+    </GridRow>
+    {backgroundColors.map((color, idx) => (
+        <GridRow key={idx}>
+            <GridCol
+                sm={12}
+                md={{ cols: 6, offset: 3}}
```
Comment (ivarni) [inline packages/ffe-grid-react/src/Grid.md]: Offsett 3 på `md` er vel litt fyfy? Syns vi skal holde oss til våre [egne standarder](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-grid-react/src/utils.dev.js#L42-L45) i dokumentasjonen :)

**Review by ivarni — DISMISSED** [review]

File: packages/ffe-grid-react/src/Grid.md (hunk 2 above)
Comment (selbekk) [reply packages/ffe-grid-react/src/Grid.md]: Uff da - fixed!

**Review by wkillerud — DISMISSED** [review]

**Comment by selbekk** [comment]
Here's a screenshot. I've added a minor improvement to the documentation so that it's easier to look at how rows and columns will look with different background colors:

image

**Review by wkillerud — APPROVED** [review]

**Pull request merged**